mmv1/products/vertexai/terraform.yaml

```diff
@@ -235,6 +235,29 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         custom_flatten: templates/terraform/custom_flatten/name_from_self_link.erb
       region: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
+  Model: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: 'projects/{{project}}/locations/{{location}}/models/{{name}}'
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "vertex_ai_model"
```

Comment:
Can you add a `_basic` test case exercising a minimal set of fields, and name this case`_advanced`?

Reply:
Does `_basic` need to also test update?